src/Pages/Saksoversikt/SakPanel.tsx

```diff
@@ -195,3 +213,70 @@ const OppgaveElement = ({ tidslinjeelement, erSist, tidslinjeOpen }: Tidslinjeel
         </div>
     );
 };
+
+const KalenderavtaleElement = ({
+    tidslinjeelement,
+    erSist,
+    tidslinjeOpen,
+}: TidslinjeelementProps) => {
+    if (tidslinjeelement.__typename !== 'KalenderavtaleTidslinjeElement') return null;
+
+    const { avtaletilstand, tekst, startTidspunkt, sluttTidspunkt, lokasjon, digitalt } =
+        tidslinjeelement as KalenderavtaleTidslinjeElement;
+    return (
+        <div className="tidslinje-element">
+            <Detail className="tidslinje-element-tidspunkt">
+                {dateFormat.format(new Date(startTidspunkt))}
+            </Detail>
+            <div className="tidslinje-element-ikon">
+                {avtaletilstand === KalenderavtaleTilstand.ArbeidsgiverHarGodtatt ? (
+                    <KalenderavtaleIkon />
+                ) : (
+                    <KalenderavtaleUtgårIkon />
```

Comment:
Hadde ikke fått med meg at det var et eget ikon også. 
Ser i skissene at det er grået ut for avlyst. Kanskje det bare er `KalenderavtaleTilstand.Avlyst` som skal ha det. 

Jeg får fikse det i widgeten også 😅